ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -497,7 +534,7 @@ objects:
         while kill -0 $openshift_install 2> /dev/null; do
             sleep 60
             echo "Polling libvirt for network, attempt #$((++i))"
-            libvirt_network=$(mock-nss.sh virsh --connect "${remote_libvirt_uri}" net-list --name | grep "${NAMESPACE}-${JOB_NAME_HASH}" || true)
+            libvirt_network=$(mock-nss.sh virsh --connect "${remote_libvirt_uri}" net-list --name | grep "${LEASED_RESOURCE}" || true)
```

Comment:
Use `CLUSTER_NAME` here?

Reply:
I did in a previous iteration, but it's a trap! :)

You see, when the terraform script runs to create the actual resource names, it gets capped off at a character could that is less than the length of ${CLUSTER_NAME}. By forcing it to ${LEASED_RESOURCE} we get around this and can guarantee uniqueness.

Reply:
I discussed this with @mhamzy https://coreos.slack.com/archives/C017UFPQA4X/p1608664950275400?thread_ts=1608567848.254300&cid=C017UFPQA4X

Reply:
How about `${CLUSTER_NAME::20}`, with 20 being replaced with whatever is sufficiently long for uniqueness but sufficiently short to avoid truncation in Terraform?

Reply:
That's a fair point, I can look into the strlen used by the terraform names.

Reply:
Strangely, the references I see for cluster_id, which is used to create the variables is 27
https://github.com/openshift/installer/blob/06f77681b6a26421f12fc3a6a783da96918a0d3e/pkg/asset/installconfig/clusterid.go#L44
https://github.com/openshift/installer/blob/06f77681b6a26421f12fc3a6a783da96918a0d3e/data/data/libvirt/main.tf#L47

The reason I mention this is strange is because in my previous test, the length of the name I passed in was:
`libvirt-ppc64le-1-0-123456`, that is an exact length of 27. I wonder if libvirt truncates the name of resources further?

Reply:
I've got it now:
So it turns out we generate a random infra id of length randomlen
https://github.com/openshift/installer/blob/06f77681b6a26421f12fc3a6a783da96918a0d3e/pkg/asset/installconfig/clusterid.go#L44

This is hardcoded to length 5, but we prepend it with a '-' so it's at least 6 characters.

This means for a full length of 27 characters, we have 21 characters to spare.
https://github.com/openshift/installer/blob/06f77681b6a26421f12fc3a6a783da96918a0d3e/pkg/asset/installconfig/clusterid.go#L60

Reply:
Implemented! :)